### ISSUE TITLE
After selecting a Learning Activities, I can see its detail

### ISSUE THREAD

**Issue by pedro16v** [catalpainternational/canoe#457]
## Conditions of Satisfaction
* I can see a reference and an estimated duration
* I have access to different sections of the Learning Activities:
  * Plan — this will be the default view when entering the Learning Activities
  * Teach
  * Extend

## Resources
* Engineering estimate: Days (2)
* Reference design: [Learning Activities - Detail](https://projects.invisionapp.com/share/6V102V1VURS9#/screens/443710125)

**Comment by pedro16v** [comment]
Hi @md8n @PeteCoward. 

This issue represents the skeleton for the Learning Activity detail page (the one with the three tabs — Plan, Teach, and Extend). I'll use this on to register our clarification on fields, headers, types of content and such, as decribed in the [support document Lee created](https://docs.google.com/spreadsheets/d/1sIw88Jh3ejsBkLRXujGFgshNqw0wmKJtL_3kzvnG6E4/edit#gid=1936780302).

I think previous pages (list of learning activities) are straightforward so I'll focus on the complexity of the detail page only.

After running our thought through @KaraChesal again, the conclusion is rather simple and is reflected in the "Learn activity details" spreadsheet tab of the above mentioned document.

In summary:
- Section and header titles are fixed and mandatory
- The exception is the "Formative Assessment" section that still needs design and we might end up pushing to a next release.
- Section and header titles need to be translatable.
- The content that can go in each section may vary. **However, Kara is happy to go with the same options for every section if that makes it simpler and faster at this point.**
